### ISSUE TITLE
Ask for more information from users

### ISSUE THREAD

**Issue by loorm** [citizenos/citizenos-fe-old#521]
**What is the user story?**
Activist authors need to get some contacts from users of their topics, especially for social-ID signing.

**What is the requested feature?**
Ask for people to provide their contacts, where they would prefer to be contacted about the topic they have just signed. This input- and consent form would be displayed every time someone has voted, unless they have given us their contacts and consent already.

Alternatively, there might be a larger form under profile, that we invite people to fill in, but do not mandate.

**Additional information.**
This information could then be made available to the authors upon downloading the signed container or be kept with us, but authors could send e-mail to them via our system.

**Comment by loorm** [comment]
This is related to the notification system user epic in Big Picture Board (and had been linked there), because in order to receive notifications, you need to provide contact information, where you want the notifications to arrive.

This definitely needs a review from @kativellak, however, I propose the following:

- Expand the contact info part in "My account". Currently we only have e-mail. I would add SMS/phone number and secondary e-mail (I want to be invited to topics with primary e-mail, but would like to receive notifications somewhere else). 
- Add a permissions page, where I can mark, which notifications do I allow in which channels and whether or not I want to make my e-mail visible to everyone / users I share topics with.

After Kati has given her opinion about this, the issue can move on to design and copy.

**Comment by anettlinno** [comment]
@loorm if I remember correctly then the user story for this was following:
As a resident of Harku parish I want to initiate public discussion, vote over the topic and then send it  to the local government. I want that local government would be able to contact (and notify) everyone who participated in this initiative about further updates. Therefore people should have an opportunity to leave their contacts (optional). 

I think we can do it before we fully develop the notification system and include this there :)

**Possible solution:**
We now have "Preferences" window under "My account" and before the notification system is ready, users could have an opportunity to choose either they want to share their phone number and/or e-mail address with the admin user or not. 
This information could be published together with voting results in csv file.
 
In short:
- Adding two more rows under "Profile": one for phone number and second for alternative email.
- Under the "Preferences" users could have an opportunity to choose either they want to share their e-mail and/or phone with the topic initiator. Maybe something like that: "I consent to share my email address with the topic initiator" (and the same with phone number) ON/OFF. We should also mention that if they do so then COS is no longer responsible for the usage of their personal data. 

Definitely needs @kativellak input here :)

**Comment by KatiVellak** [comment]
@anettlinno please let's have a discussion about this.

**Comment by anettlinno** [comment]
Triage 49. Discussed and decided that we do not create separate solution for asking and providing contacts. We will solve this issue inside the notification system. 

Topic author should take the responsibility of gathering information from local government and uploading this to follow-up feed. Once the notification system is developed, all participants will have an opportunity to receive notifications via the channel they have chosen. 

This issue is tagged to the notification system user epic in Big Picture Board.

**Comment by ilmartyrk** [comment]
Moved to https://github.com/citizenos/citizenos-fe/issues/51
